common/models/role.js

```diff
@@ -121,7 +121,11 @@ module.exports = function(Role) {
    * Add custom handler for roles.
    * @param {String} role Name of role.
    * @param {Function} resolver Function that determines if a principal is in the specified role.
-   * Signature must be `function(role, context, callback)`
+   * Signature could be using callback: `function(role, context, callback)`
+   * or returning promise:
+   * `function(role, context) {
+   *    return new Promise(function(resolve, reject){});
+   *  }`
```

Comment:
A function that determines if a principle is in the specified
role. Should provide a callback or return a promise.